### PR TITLE
change in build.sh

### PR DESCRIPTION
**Pull request by mshariq-nerd**
build.sh with help, version usage info; and license etc

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -20,35 +54,82 @@ show_files_with_size () {
...
+    echo "${USAGE}"
+
+  elif [ "$OPTION" == "VERSION" ]
+  then
+    echo $__version__
+
+  elif [ "$OPTION" == "START" ]
+  then
+    build
+
```
Comment (ashish-jabble) [inline build.sh]: Remove `START`?

File: build.sh [hunk 2]
```diff
@@ -20,35 +54,82 @@ show_files_with_size () {
...
-RELEASBLE_BUILD="foglamp-gui-${VERSION}.tar.gz"
-echo "Creating compressed build artifacts for release ..." 
-tar -zcvf ${RELEASBLE_BUILD} dist &>/dev/null
-echo "Created ${RELEASBLE_BUILD}"
-echo "Done."
+    execute_command
+  done
+else
+  OPTION="START"
+  execute_command
```
Comment (ashish-jabble) [inline build.sh]: else ?

File: build.sh [hunk 3]
```diff
@@ -20,35 +54,82 @@ show_files_with_size () {
...
+    T_SIZE=$(du -hs dist)
+    echo -e INFO " ${CWARN} Size: ${T_SIZE} ${CRESET}" 
+
+    echo -e INFO: "${CINFO} Removing unwanted contents ... ${CRESET}"
+    remove_unwanted_files
+
+    T_SIZE=$(du -hs dist)
+    echo -e INFO: "${CINFO} Deployable dist size ${CWARN}  ${T_SIZE} ${CRESET}" 
+
+    RELEASBLE_BUILD="foglamp-gui-${__version__}.tar.gz"
```
Comment (ashish-jabble) [inline build.sh]: `RELEASABLE_BUILD`

File: build.sh (hunk 2 above)
Comment (praveen-garg) [reply build.sh]: when run without args, start building

File: build.sh (hunk 1 above)
Comment (praveen-garg) [reply build.sh]: no, we are setting this as default option

File: build.sh (hunk 2 above)
Comment (praveen-garg) [reply build.sh]: but yes, we can simply call `build` here and remove `elif [ "$OPTION" == "START" ]` block /cc @mshariq-nerd

File: build.sh [hunk 4]
```diff
@@ -99,8 +107,7 @@ execute_command() {
   then
     echo $__version__
 
-  elif [ "$OPTION" == "START" ]
-  then
+  else
```
Comment (praveen-garg) [inline build.sh]: keep this only with if / elif checks

File: build.sh [hunk 5]
```diff
@@ -130,6 +137,5 @@ then
     execute_command
   done
 else
-  OPTION="START"
   execute_command
```
Comment (praveen-garg) [inline build.sh]: call build here not as another else  in execute_command

File: nginx.conf [hunk 6]
```diff
@@ -13,25 +13,29 @@ http {
...
         location / {
             try_files $uri$args $uri$args/ /index.html;
         }
 
-        # serve static content
-        location /static/ {
-            # path for static files
-            root .;
-        }
+        # remove this?!
```
Comment (praveen-garg) [inline nginx.conf]: I think we should remove this or find a way for dynamic proxy?
As we can have call to 0.0.0.0:1995 via https or as per configured port

**Comment by mshariq-nerd** [comment]
fixed per feedback.

**Pull request merged**